core/preferences/src/main/java/de/mm20/launcher2/preferences/migrations/Migration_12_13.kt

```diff
@@ -10,5 +11,10 @@ class Migration_12_13: VersionedMigration(12, 13) {
                     .setDatePart(true)
                     .build()
             )
+            .setResultOrdering(
```

Comment:
Default value should also be added to `Defaults.kt` since migration only run for existing users.